venafi/resource_venafi_policy.go

```diff
@@ -18,13 +18,13 @@ func resourceVenafiPolicy() *schema.Resource {
 		Exists: resourceVenafiPolicyExists,
 
 		Schema: map[string]*schema.Schema{
-			"zone": &schema.Schema{
```

Comment:
same as on certificate resource, the schema.Schema references are being removed...